examples/gno.land/r/gnoland/valopers/valopers.gno

```diff
@@ -0,0 +1,468 @@
+// Package valopers is designed around the permissionless lifecycle of valoper profiles.
+package valopers
+
+import (
+	"crypto/bech32"
+	"errors"
+	"regexp"
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/avl/pager"
+	"gno.land/p/demo/combinederr"
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	ErrValoperExists       = errors.New("valoper already exists")
+	ErrValoperMissing      = errors.New("valoper does not exist")
+	ErrAddressExists       = errors.New("valoper updated address exists")
+	ErrSameAddress         = errors.New("valoper updated address is the same")
+	ErrInvalidAddress      = errors.New("invalid address")
+	ErrOriginCaller        = errors.New("origin call is not the same")
+	ErrInvalidMoniker      = errors.New("moniker cannot be empty")
+	ErrInvalidDescription  = errors.New("description cannot be empty")
+	ErrInvalidPubKey       = errors.New("invalid public key")
+	ErrInvalidP2PAddress   = errors.New("invalid P2P address")
+	ErrP2PAddressesMissing = errors.New("P2P addresses cannot be empty")
+)
+
+var (
+	valopers     *avl.Tree                  // valopers keeps track of all the active validator operators. Address -> Valoper
+	instructions string                     // markdown instructions for valoper's registration
+	minFee       int64     = 20 * 1_000_000 // minimum gnot must be paid to register.
+	minCoin                = std.NewCoin("ugnot", minFee)
+)
+
+// Valoper represents a validator operator profile
+type Valoper struct {
+	Moniker     string // A human-readable name
+	Description string // A description and details about the valoper
+
+	Address      std.Address // The bech32 gno address of the validator
+	PubKey       string      // The bech32 public key of the validator
+	Active       bool        // Flag indicating if the valoper is active
```

Comment:
rename this to something else; this flag should be about the owner to say wether or not they want to keep the validator running or not; while the Active flag is something that depends on govdao's management of the active valset.

Reply:
Renamed Active to KeepRunning.
https://github.com/gnolang/gno/pull/3830/commits/8f0d0cf33d605de05e916b2430806099c9301c26